tests/console/soundtouch.pm

```diff
@@ -18,9 +18,23 @@ use strict;
 use warnings;
 use testapi;
 use utils;
+use registration qw(cleanup_registration register_product add_suseconnect_product get_addon_fullname remove_suseconnect_product);
+use version_utils 'is_sle';
 
 sub run {
     select_console 'root-console';
+    # development module needed for dependencies, released products are tested with sdk module
+    if (get_var('BETA')) {
+        my $sdk_repo = is_sle('15+') ? get_var('REPO_SLE_MODULE_DEVELOPMENT_TOOLS') : get_var('REPO_SLE_SDK');
+        zypper_ar 'http://' . get_var('OPENQA_URL') . "/assets/repo/$sdk_repo", name => 'SDK';
+    }
+    # maintenance updates are registered with sdk module
+    elsif (get_var('FLAVOR') !~ /Updates|Incidents/) {
+        cleanup_registration;
+        register_product;
+        add_suseconnect_product('sle-module-desktop-applications');
```

Comment:
this is valid only on SLE15+

Reply:
and 12SP5 in development when it's not BETA anymore